.github/workflows/release-create.yml

```diff
@@ -0,0 +1,147 @@
+
+# No secrets (except the last step uses the GITHUB_TOKEN to push release-commits+tag to the branch)
+# are needed for this workflow.
+
+name: Create Release
+
+on:
+  # Manually triggered
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: 'The version to release - e.g. `0.5.0`'
+        required: true
+      nextVersion:
+        description: 'The nest development version - e.g. `0.6.0` - without the SNAPSHOT suffix'
+        required: true
+      ref:
+        description: 'Optional: the Git reference to create the release from. Defaults to HEAD of the main branch.'
+        required: false
+        default: ''
+
+jobs:
+  build:
+    name: Create release
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'projectnessie/nessie' }}
+    env:
+      GIT_TAG: nessie-${{ github.event.inputs.releaseVersion }}
+      RELEASE_VERSION: ${{ github.event.inputs.releaseVersion }}
+      NEXT_VERSION: ${{ github.event.inputs.nextVersion }}
+
+    steps:
+
+    ### BEGIN runner setup
+    - name: Update /etc/hosts
+      run: echo -e "$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)\t$(hostname -f) $(hostname -s)" | sudo tee -a /etc/hosts
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.events.inputs.ref }}
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    # Using Python 3.8
+    # Might be worth to bump the version and/or make it configurable in the future
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Cache local pip repository
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Cache local Maven repository
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.m2/repository
+          !~/.m2/repository/org/projectnessie
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install bumpversion
+    ### END runner setup
+
+    - name: Check commit status
+      run: |
+        gh api repos/${GITHUB_REPOSITORY}/commits/${GITHUB_SHA}/status --jq 'if (.state == "success") then "OK" else error("Commit status not OK") end'
+
+    - name: Bump Maven release version ${{ github.event.inputs.releaseVersion }}
+      run: |
+        ./mvnw versions:set -DgenerateBackupPoms=false -DnewVersion=${RELEASE_VERSION}
+
+    - name: Bump Python release version ${{ github.event.inputs.releaseVersion }}
+      working-directory: ./python
+      run: |
+        bumpversion --no-commit --new-version ${RELEASE_VERSION} minor
```

Comment:
we ahve to test how this will work if its a patch release or a major release (eg not minor). I don't know how bumpversion behaves

Reply:
You mean for a `0.x.x` to `1.x.x` bump?
TBH, I've no clue what that `minor` thing actually means since we specify the `--new-version` option. I think that "part" (`minor`) has a meaning when we'd specify the `--current-version` option.

Reply:
cool, I believe we can drop the `minor` then since we are specifying the release

Reply:
IIRC not. `bump2version` requires an argument, so I suspect, that we have to keep `minor` :(

Reply:
ok, it looks like its just ignored. We may alos want to add `--no-tag` as by default it creates a tag also.